src/Analyzers/CSharp/CodeFixes/InvokeDelegateWithConditionalAccess/InvokeDelegateWithConditionalAccessCodeFixProvider.cs

```diff
@@ -109,7 +109,9 @@ public override Task RegisterCodeFixesAsync(CodeFixContext context)
                 newStatement = block.WithStatements(SyntaxFactory.SingletonList(newStatement));
             }
 
-            newStatement = newStatement.WithAdditionalAnnotations(Formatter.Annotation);
+            newStatement = newStatement.WithAdditionalAnnotations(Formatter.Annotation)
+                .WithTrailingTrivia(ifStatement.GetTrailingTrivia());
```

Comment:
this might need to append, as opposed to 'With'.  As thsi might blow away trivia that hte user had before which we wanted to keep, right?

Reply:
Can do. Do we have any helpers for skipping whitespace trivia when doing this? Otherwise we'll get double new-lines everywhere.

Reply:
Couldn't find one.. let me know if you think its worth adding to `SyntaxTriviaListExtensions`

Reply:
> Couldn't find one.. let me know if you think its worth adding to SyntaxTriviaListExtensions

It really depends.  if it's something you will use in several *differnt* features, sure.  But in general, i don't like one-off extensions that aren't used in multiple places getting put into our Extensions types.

Reply:
Yeah, thats fair (and I agree). I think I was tripped up by EndOfLine not being whitespace and there is a `SkipLeadingWhitespace()` extension method.

Though having said that, auto-merge has made the conversation moot it in this case :)